### ISSUE TITLE
Request for fix: MissingFieldInEquals many hits with UnsupportedOperationException

### ISSUE THREAD

**Issue by stokpop** [jborgers/PMD-jPinpoint-rules#61]
The MissingFieldInEquals should not flag for fields of classes that implement the following "UnsupportedOperationException" pattern.

```java
@SuppressFBWarnings("EQ_UNUSUAL")
public final boolean equals(Object other) {
   throw new UnsupportedOperationException("equals not designed.");
}
 
public final int hashCode() {
   throw new UnsupportedOperationException("hashCode not designed.");
}
```

**Comment by jborgers** [comment]
Fixed
